components/mosart/src/riverroute/RtmMod.F90

```diff
@@ -1790,12 +1803,24 @@ subroutine Rtmini(rtm_active,flood_active)
 
     call t_stopf('mosarti_mosart_init')
 
+    call t_startf('mosarti_sediment_init')
+    if(sediflag) then
+        call MOSART_sediment_init(rtmCTL%begr, rtmCTL%endr, rtmCTL%numr)
+    end if
+    call t_stopf('mosarti_sediment_init')
+
     if (wrmflag) then
        call t_startf('mosarti_wrm_init')
        if (wrmflag) then
           call WRM_init()
        endif
        call t_startf('mosarti_wrm_init')
+
+       call t_startf('mosarti_reservoir_init')
+       if (wrmflag) then
+          call MOSART_reservoir_init()
```

Comment:
The name MOSART_reservoir_init() is a little confusing because WRM_init is also related to reservoir.

Reply:
renamed to    "MOSART_reservoir_sed_init()"